### DISCUSSION TITLE
Donate

### DISCUSSION THREAD

**Discussion by KiranKowlgi** [denisneuf/python-amazon-ad-api#39] [General]
Do you have an alternate method for donation? I just tried to donate and it gives me this error

Donations to this recipient aren't supported in this country

**Comment by denisneuf** [comment]
Hi @KiranKowlgi thanks for your support, did you tried the PayPal link?
https://www.paypal.com/donate?hosted_button_id=G3KB6M2G9YV9C
Not sure from which country are you trying and if there is some restrictions : )

**Reply by KiranKowlgi** [reply]
Trying to send it from India and it looks like it just won't go through.. let me know if you have an alternative and thanks again!!

**Reply by denisneuf** [reply]
I don't have for now another alternative method but I appreciate the intention.

**Comment by denisneuf** [comment]
Hi @KiranKowlgi if want to send some support, could you try the gitgub sponsorship?
[https://github.com/sponsors/denisneuf](https://github.com/sponsors/denisneuf)
I just set-up and would like to check if everything is ok.
Thank you.

**Reply by KiranKowlgi** [reply]
Yep perfect that looks like it went through :)
